test/client/fixtures/sandbox/fetch-test.js

```diff
@@ -410,6 +411,12 @@ if (window.fetch) {
         });
 
         module('should emulate native behaviour on headers overwriting', function () {
+            // NOTE: These tests fail in the Android 6.0 browser with the following errors:
+            // Failed to construct 'Request': Cannot construct a Request with a Request object that has already been used.
+            // Failed to execute 'fetch' on 'Window': Cannot construct a Request with a Request object that has already been used.
+            if (browserUtils.isAndroid)
```

Comment:
We usually use browser-related conditions before test.
So, change condition and comment
```js
// NOTE: In Android 6.0 it is impossible to initialize Request using another Request as parameter.
// https://developer.mozilla.org/en-US/docs/Web/API/Request/Request
// https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
if (!browserUtils.isAndroid) {
    module('should emulate native behaviour on headers overwriting'
}
```